packages/bsky/src/data-plane/server/routes/profile.ts

```diff
@@ -102,6 +114,7 @@ export default (db: Database): Partial<ServiceImpl<typeof Service>> => ({
         priorityNotifications: row?.priorityNotifs ?? false,
         trustedVerifier: row?.trustedVerifier ?? false,
         verifiedBy,
+        statusRecord: status,
```

Comment:
Perhaps we could add the new tag fields here, mostly for self-documentation purposes.